src/biokbase/narrative/jobs/job.py

```diff
@@ -369,3 +436,82 @@ def dump(self):
         """
 
         return {attr: getattr(self, attr) for attr in [*ALL_JOB_ATTRS, "_last_state"]}
+
+    # def _create_error_state(
```

Comment:
is this handled somewhere else now?

Reply:
That was the one in jobmanager.py that was never used. It's called in _construct_job_status when the `state` is `None`. But I think that was never the case. Maybe I should double check and then just delete this and not code hoard.

Reply:
Or should I add that error handling back in? I think the `state` was always coming from ee2, so it normally wouldn't be `None`. But am I whittling down too much

Reply:
It looks like this code path is for when ee2 didn't return any information about the job. It might be more appropriate to have it in the job manager module, though.

Reply:
If you can verify that the state passed to `_contruct_job_status` will never be none, then just delete this, I think.

There was a case in some older version where the job lookup could fail and just return None, so that was put in to avoid something that would be catastrophic. I think that's all handled better now.